tests/ImageSharp.Tests/TestUtilities/ImageProviders/ITestImageProvider.cs

```diff
@@ -25,29 +27,32 @@ public interface ITestImageProvider
     /// <summary>
     /// Provides <see cref="Image{TPixel}" /> instances for parametric unit tests.
     /// </summary>
-    /// <typeparam name="TPixel">The pixel format of the image</typeparam>
+    /// <typeparam name="TPixel">The pixel format of the image.</typeparam>
     public abstract partial class TestImageProvider<TPixel> : ITestImageProvider
```

Comment:
The file name is misleading now, we should either always move classes to a separate files, or change the rules.

Reply:
In this instance, yes I'd probably move. The relaxing of the rule is designed to allow things like combining enums and classes in the same file

Reply:
I have moved the interface to a separate file